src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export class Actions {
```

Comment:
Yes, sorry, that probably was not very clear. I did change the labels inside the ion-button, but the `segment` variable that keeps track of which segment is currently selected receives a string from the ion-segment component, so when I compare it to the values of the enum inside the switch case statement, it always equals false because it compares a string with a number and it compares using `===`.